### PR TITLE
NOC email template + deleteModal translations

### PR DESCRIPTION
**Pull request by minsunny**
nanopay : nanoPayinc/NANOPAY#11878

https://nanopay.atlassian.net/browse/NP-3407
https://nanopay.atlassian.net/browse/NP-3564

added NOC template - NOC email notifications was used without no template, added basic templated to set its body and subject properly
DeleteModal translation

### CODE REVIEW

File: src/foam/nanos/notification/EmailSetting.js [hunk 1]
```diff
@@ -91,13 +91,12 @@ foam.CLASS({
...
           }
-        }    
+        }
 
         try {
           if ( foam.util.SafetyUtil.isEmpty(notification.getEmailName()) ) {
-            message.setSubject(notification.getTemplate());
             message.setBody(notification.getBody());
-            EmailsUtility.sendEmailFromTemplate(x, null, message, null, null);
+            EmailsUtility.sendEmailFromTemplate(x, null, message, "NOC", null);
```
Comment (jlhughes) [inline src/foam/nanos/notification/EmailSetting.js]: This is just plain wrong.  If an email template is not provided, then emails are not generated. 
You can generate a warning if you like.. 
so in this first part do not call sendEmailFromTemplate.

File: src/foam/nanos/notification/EmailSetting.js [hunk 2]
```diff
@@ -91,18 +93,19 @@ foam.CLASS({
           if ( config != null ) {
             notification.getEmailArgs().put("link", config.getUrl());
           }
-        }    
+        }
+
+        if ( "NOC".equals(notification.getEmailName()) && ! SafetyUtil.isEmpty(notification.getBody()) ) {
```
Comment (minsunny) [inline src/foam/nanos/notification/EmailSetting.js]: NOC notification doesnt set an email args for now and some notifications not storing its body after notification translation stuffs updated. So temporarily added the condition if "NOC".equals(notification.getEmailName()) but in the near future when going through this
https://nanopay.atlassian.net/browse/NP-3353, might be considered here.

File: src/foam/nanos/alarming/AlarmLoggingDAO.js [hunk 3]
```diff
@@ -48,6 +48,7 @@ foam.CLASS({
           if ( alarm.getIsActive() ) {
             Notification notification = new Notification.Builder(x)
               .setTemplate("NOC")
+              .setEmailName("NOC")
```
Comment (jlhughes) [inline src/foam/nanos/alarming/AlarmLoggingDAO.js]: This is on the template.  It's not needed here. that's the whole point of the notification template.

File: src/foam/nanos/notification/EmailSetting.js (hunk 2 above)
Comment (jlhughes) [reply src/foam/nanos/notification/EmailSetting.js]: I don't care, simply remove all of this 'NOC' testing on the foam side. 
It's not the foam side's responsibility.

**Review by jlhughes — APPROVED** [review]

**Pull request merged**